pingcap/tidb-operator
# add aws ami version link

### What problem does this PR solve? 
fix #1607
### What is changed and how does it work?
Add aws ami version link to the variable description so that users can know exactly which version to set if they want to.
### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
   - `terraform init`, `terraform validate`, `terraform apply` works as expected.

Code changes

 - Has Terraform scripts change

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```
